walterrowe/magick-frames
# add script to create defined style droplets

Add applescript to create defined style droplets

- update README with instructions
- rename droplet script to StyleDroplet
- create CreateStyleDroplets script
    - choose folder for created droplets
    - choose source droplet script
    - create droplets in selected folder
    - show progress bar while creating
    - notify how many droplets created
    - open droplets folder with Finder

See [Mac Automation Scripting Guide - Display Progress](https://developer.apple.com/library/archive/documentation/LanguagesUtilities/Conceptual/MacAutomationScriptingGuide/DisplayProgress.html).
